### ISSUE TITLE
JPype support

### ISSUE THREAD

**Issue by Thrameos** [tcalmant/python-javaobj#28]
Not sure it this is any help, but I added support for pickling of Java objects to JPype.  I was looking for solutions to issues during that effort and ran across this library.  However, I ultimately ended up using a pure Java solution to achieve the required effect largely because writeObject overloaded based Java classes were too difficult to support from within Python directly.   That and the fact that ObjectOutputStream is not an an interface made it hard to go any other direction.  I know there are significant differences between using Python to pickle Java and a library to read and write serialization files from within Python, but I felt I should keep this effort informed.

**Comment by tcalmant** [comment]
That's good to know :)

Note that the main purpose of javaobj is to read data written from "foreign" Java software without requiring the execution, not even the installation, of a JVM.

**Comment by Thrameos** [comment]
Yeah I figured as such.  I took one look at the internals of ObjectReader and concluded it badly needed a rewrite.  There was nothing reusable in the thing (not to mention the "it pretends to be an interface" design).  I can see why a library to manipulate the object serialization stream would be required given the lack of facilities that interface provides.

**Comment by tcalmant** [comment]
Closing as no action needed.
